docs/websites/hosting/set-up-web-server-host-website/index.md

```diff
@@ -96,7 +96,7 @@ This configuration is sufficient to get you started. For more advanced options a
 
 Other sites, such as [WordPress](/docs/websites/cms/install-wordpress-ubuntu-18-04/), need a database in addition to a web server. This combination is known as a **stack**. WordPress is often used with the extremely popular LAMP stack (Linux, Apache, MariaDB and PHP). To install a LAMP stack manually, find the guide for your distribution in our [LAMP](/docs/web-servers/lamp/) section.
 
-If you are using WordPress, another option is to use Docker. All of the components needed to run WordPress, along with WordPress itself, are bundled into a container that can be deployed with single command. Official Docker images are also available for other CMS platforms including [Ghost](https://hub.docker.com/_/ghost/) and [Joomla](https://hub.docker.com/_/joomla/). <!--- See our [WordPress on Docker](/docs/quick-answers/install-wordpress-using-docker/) guide for details. ---->
```

Comment:
The HTML comment here had an extra `-` in the opening `<!---`, so Hugo actually rendered it, and the link inside was broken. Not sure why this was not an issue previously.

I took this as an opportunity to link to our WordPress+Docker Compose guide